indico/modules/events/sessions/templates/management/session_blocks.html

```diff
@@ -0,0 +1,20 @@
+<table class="i-table-widget">
+    <thead>
+        <tr class="i-table">
+            <th class="i-table">{% trans %}Title{% endtrans %}</th>
+            <th class="i-table">{% trans %}Start date{% endtrans %}</th>
+            <th class="i-table">{% trans %}Duration{% endtrans %}</th>
+            <th class="i-table">{% trans %}Number of contributions{% endtrans %}</th>
```

Comment:
a bit long, no? especially considering that the column will only contain a somewhat short number